src/components/buttons/ButtonOutlined.tsx

```diff
@@ -0,0 +1,13 @@
+import { Button } from "@material-ui/core"
+interface ButtonOutlinedProps {
+  text: string
+  onclick: () => void
+}
+
+export const ButtonOutlined = (props: ButtonOutlinedProps) => {
```

Comment:
e làm như này thì mỗi khi dùng props khác của Material Button thì lại phải khai thêm trong ButtonOutlinedProps
=> mỗi lần thêm prop mới lại phải sửa
```suggestion
export const ButtonOutlined = styled(Button).attrs((props) => ({ variant: "outlined", ...props }))``
```
như này thì mình có thể dùng component như Button của Material UI